dcos_test_utils/aws.py

```diff
@@ -502,24 +502,24 @@ def get_host_ips(self):
 
 
 OS_AMIS = {
-    'cent-os-7': {'ap-northeast-1': 'ami-965345f8',
```

Comment:
" I think aws and gce should have the same os_name options and then each maintain a mapping of os_name -> instance_ami or os_name -> source image "  -> I thought that meant having the same possible values for os_name for both gce and aws